### DISCUSSION TITLE
Amendments to extend Deadline Receipt Requests for expired DPS

### DISCUSSION THREAD

**Discussion by EvijaMozga** [OP-TED/eForms-SDK#975] [Q&A] [answered]
Dear OP-TED Team,

We would like to find out if it is possible to submit amendments to the Contract Notice, if BT-766 is "Dynamic Purchasing System" and the deadline for BT-1311 "Deadline Receipt Requests" specified in the originally published contract notice has passed. The CA with the amendment notice is planning to extend the validity period of the Dynamic Purchasing System.

Thank you in advance!

Kind regards.
The Procurement Monitoring Bureau

**Comment by YvesJo** [comment, marked as answer]
Hi,
For this Business and non - SDK related question, please get in touch directly with the helpdesk.
KR

**Comment by EvijaMozga** [comment]
Hello,

How can we reach the helpdesk?
Thank you!

Kind regards,

Evija Mozga
Head of Information Department
Phone (+371) 20118597
E-mail: ***@***.******@***.***>

Procurement Montoring Bureau of Latvia
Smilšu street 1, Riga, LV-1919, Latvia
Website: www.iub.gov.lv<http://www.iub.gov.lv/>
E-mail: ***@***.******@***.***>
Facebook: Iepirkumu uzraudzibas birojs<https://www.facebook.com/IUB.GOV.LV/>
***@***.***

**Reply by YvesJo** [reply]
You may use the following online form: https://ted.europa.eu/en/contact
